shell/client/shell.html

```diff
@@ -1078,6 +1070,51 @@ <h4 class="share-with-others">Share with others</h4>
       <p>To get support for LDAP, SAML, and organization management, enable
         <a href="{{featuresPath}}">Sandstorm for Work</a>.</p>
     {{/if}}
+    <div class="legacy-email-config">
+      <h4>SMTP Configuration: <button id="admin-settings-send-toggle">Test</button></h4>
```

Comment:
Mmmm, it's partly about keeping identity service checkboxes together, and partly about keeping identity management and org management (which ties in deeply with identity management) together.  On the whole, I think that whether a feature is enabled by Sandstorm for Work is a less-interesting grouping to users than the things that it interacts with.

What is clear is that email relay configuration doesn't really belong here, so it's good that the new admin UI separates this out.

If anything, perhaps LDAP and SAML don't need their own `<h4>`, or things that are Sandstorm-for-Work-exclusive features could be made to have a different background color.  But I think that's not really important here, and I'm disinterested in spending much more time on a design that I'm aiming to replace in the near term. :P

I intend to leave this one as-is.